### ISSUE TITLE
Alter table no longer works after updating to beta-20170413 

### ISSUE THREAD

**Issue by abronte** [cockroachdb/activerecord-cockroachdb-adapter#4]
Looks like the latest 20170413 beta broke any migrations that alter a table. Looks like it might be due to this PR https://github.com/cockroachdb/cockroach/pull/14368.

Haven't found a work around yet. Rolling back to the 20170330 version alter table worked.

Error:
```
PG::InternalError: ERROR:  statement cannot follow a schema change in a transaction
: INSERT INTO "schema_migrations" ("version") VALUES ($1) RETURNING "version"
```

Cockroach version:
```
CockroachDB node starting at 2017-04-14 04:25:24.652071187 +0000 UTC
build:      CCL beta-20170413-dirty @ 2017/04/13 21:06:17 (go1.8.1)
admin:      http://12c3b6ed5083:8080
sql:        postgresql://root@12c3b6ed5083:26257?sslmode=disable
logs:       /data/logs
store[0]:   path=/data
status:     restarted pre-existing node
clusterID:  cc833425-15bf-4781-be38-c9934e24140f
nodeID:     1
```

**Comment by jordanlewis** [comment]
Hi @abronte,

Thanks for the report. We recently changed our transaction semantics to disallow schema changes that are not the final statement of the transaction, because we do not support transactional schema changes at this time.

It looks like you're using ActiveRecord - is that right?

It seems like we'll either have to relax this restriction somewhat or modify the ActiveRecord adapter to stop doing its DDLs transactionally.

**Comment by abronte** [comment]
Yup I'm using activerecord 5.0.2.

**Comment by tamird** [comment]
Fixed by #9. Thanks @abronte!
